### ISSUE TITLE
opentelekomcloud_identity_user_v3 - always trying to update email

### ISSUE THREAD

**Issue by RykHawthorn** [opentelekomcloud/terraform-provider-opentelekomcloud#2255]
# Terraform provider version
```
Terraform v1.5.4
on darwin_amd64
+ provider registry.terraform.io/opentelekomcloud/opentelekomcloud v1.34.2
```

# Affected Resource(s)
- opentelekomcloud_identity_user_v3

# Terraform Configuration Files
```hcl
resource "opentelekomcloud_identity_user_v3" "user_test" {
  name  = "test_user"
  email = "<REDACTED>"

  send_welcome_email = true
}
```

# Debug Output/Panic Output
[terraform output](https://gist.github.com/RykHawthorn/6b607d78b192a5efe34413ee39d51685)
[terraform trace log](https://gist.github.com/RykHawthorn/5f8307bbabafd6113019771654d48b70)

# Steps to Reproduce
1. `terraform apply`
2. `terraform apply` again

# Expected Behavior
The 1. `terraform apply` has created the user and sent a welcome email. The 2. `terraform apply` finds no changes to the plan.

# Actual Behavior
The 1. `terraform apply` has created the user and sent a welcome email. The 2. `terraform apply` finds changes to the `email` and does an `update in-place` regardless that the email didn't actually change and therefore the welcome email will also be resend.

# Important Factoids
This behavior seems to be OK in version 1.30.1 and was introduced in version 1.30.2.

# References
- #1838

**Comment by artem-lifshits** [comment]
Hello @RykHawthorn I was able to reproduce the issue, thank you for reporting.
The API for user management was changed, I will update the resource.

**Comment by artem-lifshits** [comment]
@RykHawthorn please check latest release.

**Comment by RykHawthorn** [comment]
@artem-lifshits I can confirm that the problem is solved in provider version `v1.35.5`. Thank you 👍
